ods_ci/tests/Resources/Files/user-workload-monitoring-config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: user-workload-monitoring-config
+    namespace: openshift-user-workload-monitoring
+data:
+    config.yaml: |
+        prometheus:
+            logLevel: debug
+            retention: 15d #Change as needed
+            volumeClaimTemplate:
+                spec:
+                  resources:
+                    requests:
+                      storage: 40Gi
```

Comment:
same comment as https://github.com/red-hat-data-services/ods-ci/pull/1003/files#r1379865603

Reply:
Done

Reply:
could you also delete the duplicated file?